frontend/javascripts/oxalis/view/settings/settings_view.js

```diff
@@ -1,30 +0,0 @@
-/**
- * settings_view.js
- * @flow
- */
-
-import { Tabs } from "antd";
-import * as React from "react";
-
-import DatasetSettingsView from "oxalis/view/settings/dataset_settings_view";
-import UserSettingsView from "oxalis/view/settings/user_settings_view";
-
-const TabPane = Tabs.TabPane;
-
-type Props = {
-  dontRenderContents: boolean,
-};
-
-const SettingsView = ({ dontRenderContents }: Props) => (
-  // Don't render contents to improve performance
-  <Tabs destroyInactiveTabPane className="tracing-settings-menu" defaultActiveKey="2">
-    <TabPane tab="Annotation" key="1">
-      {!dontRenderContents && <UserSettingsView />}
-    </TabPane>
```

Comment:
> I implemented an easy way to determine whether the current tab is selected (visible) or not.

Awesome!

> Should I also do this for the tabs in the right sidebar?

Let's only do it for the left sidebar for now, as this mirrors the behavior prior to this PR. We can think about the right sidebar in another iteration. My gut reaction would be to keep the components alive but somehow prohibit the update mechanism (e.g., use `shouldComponentUpdate` to only allow updates when it's visible; strictly speaking, this would also be possible for the left sidebar). In general, this would probably be a bit more involved, though.

I created a follow-up issue for this: https://github.com/scalableminds/webknossos/issues/5335